src/App.jsx

```diff
@@ -18,14 +19,38 @@ class App extends React.Component {
     this.setState({ productId });
   }
 
+  componentDidMount() {
+
+    const config = {
```

Comment:
Potentially rename config to something less generic?

Reply:
renamed to initlizationConfig, open to better variable names though